### PR TITLE
Update logo

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on https://github.com/ppy/osu-resources/pull/305/
- Addresses https://github.com/ppy/osu/discussions/21455

| Before | After |
| :---: | :---: |
| CleanShot 2024-01-25 at 10 45 01 | CleanShot 2024-01-25 at 10 44 18 |

### CODE REVIEW

**Comment by minisbett** [comment]
Is this by any chance still up to debate? I feel like it doesn't capture the spirit of osu! at all, neither the font nor the background - It just feels like an oversimplified logo.

**Comment by peppy** [comment]
> Is this by any chance still up to debate? I feel like it doesn't capture the spirit of osu! at all, neither the font nor the background - It just feels like an oversimplified logo.

thanks for your feedback. it's not up for discussion/debate right now.

**Comment by mixalisnew** [comment]
Personally i think the dot in the O is so random nad not really nice but i dont mind it either

**Comment by peppy** [comment]
thanks for your feedback. we'll give it a chance and see how it sticks 😄 .

please hold your feedback a few weeks so it can all sink in!

**Comment by peppy** [comment]
thanks for your feedback! the logo and game design is under my control so please leave it in my hands. we'll listen to feedback, but please hold your thoughts for now. this is a development PR and we don't need design feedback here for now. you can do that on reddit or twitter and we'll probably see it there 😉 .

File: osu.Desktop/osu.nuspec [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
```
Comment (Joehuu) [inline osu.Desktop/osu.nuspec:11]: The `iconUrl` line above this has a really outdated logo. Not sure if it's shown anywhere now, but here's the doc for more investigation: https://learn.microsoft.com/en-us/nuget/reference/nuspec#iconurl

**Comment by mhdimo** [comment]
The logo itself is not bad at all however 
- the fill on the cookie is "too pink"? i don't know how to explain this at all but it doesn't really match at least with the iconic logo
- the dot in the O makes me think about this: 
green-cute-alien-with-one-eye-on-white-vector-31461691
it looks somewhat childish..
- triangles not being filled is somewhat aswell sort of disturbing to see

and that's it i guess...

**Comment by bdach** [comment]
FYI I'm currently working on updating the android icons. Someone with apple devices can handle iOS.

**Comment by peppy** [comment]
@bdach I'll wait for your update on that because it will use the same base resource (1024x required)

**Comment by bdach** [comment]
Well this took a while. As usual the android ecosystem makes me want to commit unspeakable acts.

So android now has this ["adaptive icons" bojangle](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive). The point is that the user is supposed to be able to reshape the icon in any direction or whatever and it should continue to display fine. They have a [figma template](https://www.figma.com/community/file/1131374111452281708) linked that - surprise surprise - is basically unusable and you have to use android studio instead.

The thing is that there are basically four variants of the icon: the circular "fallback" for android devices pre-API 26, and then the background/foreground "layered" versions as well as the "monochrome" version which is the "adaptive" nonsense.

In the end, on a "new" device, the logo can look like so:

| | | | |
| :-: | :-: | :-: | :-: |
| image | image | image | image |

and on my older device this happens:

image

which is stupid as all hell but this is because samsung decided that it's gonna put all icons in these squircle things in the first place and that's what android studio generated _for_ me so I don't know at this point.

I dunno. There was an attempt.

@peppy here's the raw 1024x1024 asset for you to use maybe.

ic_launcher

**Comment by Tpmega9** [comment]
big L

**Comment by april83c** [comment]
I think it would look much better with the normal version of Torus [(quick mockup of what that would look like by RuthlessRex on reddit)](https://www.reddit.com/r/osugame/comments/19faqqf/the_full_new_osulazer_logo_as_on_github_what_do/kjip0x2/)

image

**Comment by mizaagi** [comment]
why did you do this to us peppy :(

**Comment by peppy** [comment]
https://github.com/ppy/osu/discussions/26706

**Pull request merged**